dapps/v2Explorer/src/screens/App.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {useEffect, useState, useCallback} from 'react';
+import {
+  SafeAreaView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  useColorScheme,
+  View,
+} from 'react-native';
+
+import 'react-native-get-random-values';
+import '@ethersproject/shims';
+
+import '@walletconnect/react-native-compat';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import useInitialization, {web3Provider} from '../hooks/useInitialization';
+import {universalProviderSession} from '../utils/UniversalProvider';
+import {ExplorerModal} from '../components/ExplorerModal';
+
+function App(): JSX.Element {
+  const isDarkMode = useColorScheme() === 'dark';
+  const [modalVisible, setModalVisible] = useState(false);
+  const [currentAccount, setCurrentAccount] = useState<string | null>(null);
+
+  // Initialize universal provider
+  const initialized = useInitialization();
+
+  const close = () => {
+    setModalVisible(false);
+  };
+
+  const getAddress = useCallback(async () => {
+    try {
+      const signer = web3Provider.getSigner();
+      const currentAddress = await signer.getAddress();
+      setCurrentAccount(currentAddress);
+    } catch (err: unknown) {
+      console.log('Error for initializing', err);
+    }
+  }, []);
+
+  useEffect(() => {
+    // NOTE: Logs to help developers debug
+    // console.log('App Initalized: ', initialized);
```

Comment:
Commented out console logs

Reply:
Leaving this one for devs

Reply:
I'd either leave these on by default or remove and let devs add whatever logs they need themselves if they're already making changes in the code. Feels like an oversight like this.